.github/ISSUE_TEMPLATE.md

```diff
@@ -1,3 +1,17 @@
+<!--
+
+Hi, thanks for raising an issue with The Guardian's website.
+
+Please note that it may not be possible for us to comment on every issue publicly,
+but we will always do our utmost to resolve your problem
```

Comment:
Nitpicking, but I would reword this section to:

`We will do our best to respond to all issues raised, but cannot guarantee to do so.`

reason being, there will be issues we cannot resolve, and others we do not want to.